### PR TITLE
Feature/backend/config

### PR DESCRIPTION
**Pull request by codecakes**
basically 
- configures app to access secrets from appengine.
- updates readme on how to run w/ bazel locally

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/33)

### CODE REVIEW

File: covidX/gae_settings.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import os
+
+from google.cloud import secretmanager
+
+
+def access_secret_key_version():
+    """
```
Comment (codecakes) [inline covidX/gae_settings.py:7]: Codacy found an issue: [No blank lines allowed after function docstring (found 1) (D202)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- covidX/gae_settings.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707&bid=20241473)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- covidX/gae_settings.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707&bid=20241473)

File: covidX/gae_settings.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import os
+
+from google.cloud import secretmanager
+
+
+def access_secret_key_version():
+    """Access the payload for the given secret version if one exists.
```
Comment (codecakes) [inline covidX/gae_settings.py:7]: Codacy found an issue: [No blank lines allowed after function docstring (found 1) (D202)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- covidX/gae_settings.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707&bid=20241473)

File: covidX/gae_settings.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (codecakes) [inline covidX/gae_settings.py:7]: Codacy found an issue: [No blank lines allowed after function docstring (found 1) (D202)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- covidX/gae_settings.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6205707&bid=20241473)

**Review by atuljain — APPROVED** [review]

**Pull request merged**